app/(main)/chat/new/page.tsx

```diff
@@ -0,0 +1,133 @@
+'use client';
+
+import { useState } from 'react';
+import { useRouter } from 'next/navigation';
+import { Button, IconButton } from '@swapr/ui';
+import { useSession } from '@/context/SessionContext';
+import { useMutation } from '@tanstack/react-query';
+import Image from 'next/image';
+import wizardSvg from '@/public/pixel-wizard.svg';
+import { Spinner } from '@/app/components';
+
+const PRESAGIO_CHAT_API_URL = process.env.NEXT_PUBLIC_PRESAGIO_CHAT_API_URL!;
+
+const createChat = async (message: string): Promise<{ chatId: string }> => {
+  const response = await fetch(PRESAGIO_CHAT_API_URL + '/api/chat', {
```

Comment:
usually API endpoint is plural, you do a post to `/api/chats`
single endpoint is used to act on already existing item, usually with an id like /api/chat/{id} where you DELETE or PUT.